packages/lit-virtualizer/src/Virtualizer.ts

```diff
@@ -710,7 +709,6 @@ export class Virtualizer {
   // a first-class feature?
 
   private _childLoaded() {
-    // this.requestRemeasure();
   }
```

Comment:
Nit: Is it worth entirely removing this since it isn't used and tracking the TODO in a Github issue instead?